app/services/metrics/defect_escape_rate/per_project.rb

```diff
@@ -1,37 +0,0 @@
-module Metrics
```

Comment:
why was it deleted? how is it related to the rename of JiraBoard?

Reply:
Is not related to this rename. I have noticed that this file is unused because DER metric is now by product non by project.

Reply:
ok for the future, especially for real projects, consider creating separate PRs (and even creating separate tickets) when tackling unrelated stuff